src/claimtrieforks.cpp

```diff
@@ -153,27 +129,25 @@ bool CClaimTrieCacheNormalizationFork::normalizeAllNamesInTrieIfNecessary(insert
             if (support.nHeight + base->nExpirationTime <= nNextHeight)
                 continue;
 
-            CSupportValue removed;
-            assert(removeSupportFromMap(it.key(), support.outPoint, removed, false));
-            expireSupportUndo.emplace_back(it.key(), removed);
+            assert(removeSupportFromMap(it.key(), support.outPoint, support, false));
```

Comment:
This has a bug in that it modifies the item we're about to insert. That other variable was there on purpose.

Reply:
Actually it isn't, because we know that we remove same support then read it under new name, no changes in value.

Reply:
But better to iterate through copy rather than reference.